cmd/tempo/main.go

```diff
@@ -171,11 +179,20 @@ func loadConfig() (*app.Config, error) {
 		if err != nil {
 			return nil, fmt.Errorf("failed to parse configFile %s: %w", configFile, err)
 		}
+
+		if configVerify {
+			if !configIsValid(config) {
```

Comment:
as said above, let's move the config verification here instead of calling it from two spots, but I believe this needs to move below the `flag.Parse()` line to take into account both file config and cli config.